### PR TITLE
Big Sur 11.1 on Late 2013 27" iMac

### PR DESCRIPTION
**Pull request by jpasqua**

### CODE REVIEW

**Comment by BenSova** [comment]
Pretty good. Two suggestions though. This screen is about the outcome not the steps to get there, so it's how well does it run what doesn't work and what does etc is what belongs in the details. Also, remove the word networks after WPA2 Enterprise because there is a limited space, especially when there is all three works, doesn't work, and unknown.

**Comment by jpasqua** [comment]
Will do. Your instructions say "WPA2 Enterprise Networks" so you may want to edit that.
Screen Shot 2021-01-01 at 1 03 46 PM

**Comment by jpasqua** [comment]
OK, updates have been made.

**Comment by BenSova** [comment]
Let me rephrase that a little bit. The idea is to add reassurance and state what works. I would note on that model that you don't need Patch Kexts, and it would just produce errors. Here's an example from what I wrote:
Screen Shot 2021-01-01 at 2 10 45 PM

**Comment by jpasqua** [comment]
Ok. How about: "Everything works and performance is the same as Catalina. WiFi, USB, and Handoff worked without patching Kexts in the post install app, which would have produced errors."

**Comment by BenSova** [comment]
Sure, you might also want to mention that if you have a fusion drive to upgrade it to an SSD otherwise performance will be horrible.

File: Compatibility/iMac14,2.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "MacName": "iMac (27-inch, Late 2013)",
+    "Author": "@bitsplusatoms",
```
Comment (BenSova) [inline Compatibility/iMac14,2.json:3]:
```suggestion
    "Author": "@bitsplusatoms",
    "Approved": ["@BenSova"],
```

**Review by BenSova — COMMENTED** [review]
Does GitHub work?

**Comment by BenSova** [comment]
Okay thanks! Time to merge!
